src/Map.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import ReactMapGL, { NavigationControl } from 'react-map-gl';
+// import DeckGL, { GeoJsonLayer, IconLayer } from 'deck.gl';
+import {
+  graphql,
+  createFragmentContainer,
+} from 'react-relay';
+import propTypes from 'prop-types';
+import MAPBOX_ACCESS_TOKEN from './config';
+import Routes from './Routes';
+
+class Map extends Component {
+  constructor() {
+    super();
+    this.state = {
+      // Viewport settings that is shared between mapbox and deck.gl
+      viewport: {
+        width: window.innerWidth,
+        height: window.innerHeight,
+        longitude: -122.41669,
+        latitude: 37.7853,
+        zoom: 15,
+        pitch: 0,
+        bearing: 0,
+      },
+      settings: {
+        dragPan: true,
+        // dragRotate: true,
+        // scrollZoom: true,
+        // touchZoomRotate: true,
+        // doubleClickZoom: true,
+        minZoom: 0,
+        maxZoom: 20,
+        minPitch: 0,
+        maxPitch: 85,
+      },
+    };
+  }
+
+  renderMap() {
+    const onViewportChange = viewport => this.setState({ viewport });
+
+    const { viewport, settings } = this.state;
+    // I don't know what settings used for,
+    // just keeping it in following format to bypass linter errors
+    console.log(settings && settings);
+
+    return (
+      <ReactMapGL
+        {...viewport}
+        mapStyle="mapbox://styles/eddyionescu/cj9btlvm2423a2sprnbgpk9r5"
```

Comment:
while we're at it, could we also add this to the config.json?

Reply:
is this fine?
import { MAPBOX_ACCESS_TOKEN, MAP_STYLE } from './config.json';

Reply:
yeah, that's good.